bridetalk_project/src/components/atoms/fillinbox/EnterText.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
**Important comment**

This is a bad React practice. You are using 3 components `EnterEmail`, `EnterPassword`, `EnterText`, which are completely identical. Replace those components with one component that accepts more props.

F.ex., `type={props.type}`. 
If you need dynamic classnames give classnames with props as well.

The same comment applies to your PrimaryButton and CancelButton components. Delete one of them and make a generic Button component that accepts props to be styled as a primary or a cancel one.